pkg/network/ovn_kubernetes.go

```diff
@@ -287,11 +287,12 @@ func renderOVNKubernetes(conf *operv1.NetworkSpec, bootstrapResult *bootstrap.Bo
 	}
 
 	IPsecMachineConfigEnable, OVNIPsecDaemonsetEnable, OVNIPsecEnable, renderIPsecHostDaemonSet, renderIPsecContainerizedDaemonSet,
-		renderIPsecDaemonSetAsCreateWaitOnly := shouldRenderIPsec(c, bootstrapResult)
+		renderIPsecDaemonSetAsCreateWaitOnly, isUserDefinedIPsecMachineConfigPresent := shouldRenderIPsec(c, bootstrapResult)
```

Comment:
Why dont you call `isUserDefinedIPsecMachineConfigInstalled` directly from here instead of going though `shouldRenderIPsec`?

Reply:
yes, right, done.